doc/examples/denoise_localpca.py

```diff
@@ -0,0 +1,120 @@
+"""
+===============================
+Denoise images using Local PCA
+===============================
+
+Using the local PCA based denoising for diffusion
+images [Manjon2013]_ we can denoise diffusion images significantly. This algorithm is
+popular as it takes into account the directional information in diffusion data.
```

Comment:
Last sentence: 'For diffusion-weighted data this algorithm is advantageous for taking into account the directional information in diffusion data.'